CIP-0164/README.md

```diff
@@ -0,0 +1,2774 @@
+   references to additional transactions.
+
+The RB chain continues to be distributed exactly as in Praos, while Leios
+introduces separate distribution mechanisms for EB headers (for rapid discovery
+and <a id="equivocation" href="#equivocation-detection">equivocation
+detection</a>), EB bodies, and their referenced transactions.
+
+Due to the voting overhead per EB, EBs should only be announced if a transaction
+cannot be included in the base RB. Empty EBs should not be announced in the
+network as they induce a non-zero cost. Note that whether an RB is full is not
+solely determined by its byte size; in particular, the per-block Plutus limits
+could be the reason a full RB cannot contain additional transactions.
+Additionally, transactions requiring higher size or Plutus execution limits
+available through proposed Leios may necessitate placement in EBs rather than
+RBs. The lower latency provided by RBs naturally incentivizes their use first,
+enabling gradual adoption of Leios capabilities.
+
+#### Step 2: EB Distribution
+
+Nodes receiving the RB header discover the announced EB and fetch its body. The
+EB body contains references to transactions. If a node does not already possess
+a transaction referenced in the EB, it explicitly requests that transaction from
+peers. The whole process of propagating EBs and referenced transactions is
+called EB diffusion. Only minimal validation is done before forwarding this data
+to ensure rapid dissemination while full validity is determined by the
+[voting committee](#step-3-committee-validation).
+
+Minimal validation includes basic structural and cryptographic integrity checks
+(VRF verification, block hash validation, and header format verification).
+
+#### Step 3: Committee Validation
+
+After the [equivocation detection period](#equivocation-detection) of
+$3 L_\text{hdr}$ slots, a voting committee of stake pools validates the EB and
+votes within a [voting period](#voting-period) $L_\text{vote}$. Committee
+members are [selected via sortition](#committee-structure) based on the slot
+number of the RB that announced the EB. A committee member votes for an EB only
+if:
+
+1. The RB header arrived within $L_\text{hdr}$,
+2. It has **not** detected any equivocating RB header for the same slot,
+3. It finished validating the EB before $3 \times L_\text{hdr} + L_\text{vote}$
+   slots after the EB slot,
+4. The EB is the one announced by the latest RB in the voter's current chain,
+5. The EB's transactions form a **valid** extension of the RB that announced it,
+6. For non-persistent voters, it is eligible to vote based on sortition using
+   the announcing RB's slot number as the election identifier.
+
+where $L_\text{hdr}$ and $L_\text{vote}$ are
+<a href="#protocol-parameters">protocol parameters</a> represented by a number
+of slots.
+
+While not strictly a required check, honest nodes should not vote on empty EBs
+as that is obviously pointless and wasteful.
```

Comment:
What does "not strictly" here imply? The formal spec **_strictly_** requires an honest node not vote for an empty EB: [`EndorserBlockOSig.txs eb ≢ []`](https://github.com/input-output-hk/ouroboros-leios-formal-spec/blob/c7f8d61865360cd16e04c7c3f0fd481c8081deff/formal-spec/Leios/Linear.lagda.md?plain=1#L135).

- Are we trying to say that a chain is valid even if it contains empty EBs but honest nodes should never vote for an empty EB?
- Or are we saying that we didn't need to include this required check in the specification but we did anyway for reasons of efficiency?